packages/react/.eslintrc.json

```diff
@@ -6,6 +6,16 @@
       "rules": {
         "react/prop-types": "off"
       }
+    },
+    {
+      "files": "src/**/*.{ts,tsx}",
+      "globals": {
+        "document": "off",
+        "window": "off"
```

Comment:
It's reasonable to disable only `document` and `window` globals as they are context dependent.